src/main/java/bio/terra/pdao/bigquery/BigQueryPdao.java

```diff
@@ -564,6 +563,16 @@ private String prefixName(String name) {
         return PDAO_PREFIX + name;
     }
 
+    private String createSoftDeletesTableName(String tableName) {
+        return PDAO_PREFIX + "_sd_" + tableName;
+    }
+
+    private Schema buildSoftDeletesSchema() {
+        List<Field> fieldList = new ArrayList<>();
+        fieldList.add(Field.of(PDAO_ROW_ID_COLUMN, LegacySQLTypeName.STRING));
+        return Schema.of(fieldList);
```

Comment:
I think you could do `Collections.singletonList(Field.of(...))` instead of constructing and adding.